### PR TITLE
Add Linux path to README

### PR DESCRIPTION
**Pull request by kian**
Fixes https://github.com/NeverSinkDev/NeverSink-Filter/issues/149

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,14 @@ The website *[filterblade.xyz](http://www.filterblade.xyz)* is the user-friendly
...
 2) Extract the files using WinRar/7zip or a similar tool.
 
-3) Paste the .filter files (or just the folder content, if you want to use a specific style, paste the files from one of the (STYLE) folders instead) into the following folder: *%userprofile%/Documents/My Games/Path of Exile*
-
-4) INGAME: Escape -> Options -> UI -> Scroll down -> Select one of the filters from the dropdown box.
+3) Paste the .filter files (or just the folder content, if you want to use a
+specific style, paste the files from one of the (STYLE) folders instead) into
+the following folder: ```%userprofile%/Documents/My Games/Path of Exile```.  On
+Linux, this directory will be within
+```~/.local/share/Steam/steamapps/compatdata/```.
```
Comment (johnsmithdoe970) [inline README.md:36]: I suppose that's only for steam though. Anyway, it's probably better than nothing.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,14 @@ The website *[filterblade.xyz](http://www.filterblade.xyz)* is the user-friendly
...
+
+4) INGAME: Escape -> Options -> UI -> Scroll down -> Select one of the filters
```
Comment (johnsmithdoe970) [inline README.md:38]: nit: Why the change on this line? The linebreak is not done on the other paragraphs above, so it's probably not worth changing it here

**Review by johnsmithdoe970 — COMMENTED** [review]
Not perfect, but a good improvement, IMO

**Comment by kian** [comment]
I don't remember the details of this PR and I'm not playing this league.  I'll pick this up again when I'm back on PoE/Neversink.
